pkg/syncer/volume_health.go

```diff
@@ -77,58 +77,82 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 		}
 	}
 
-	for _, vol := range queryResult.Volumes {
-		log.Debugf("Volume %q Health Status %q", vol.VolumeId.Id, vol.HealthStatus)
+	// volumeIdToHealthStatusMap maps vol.VolumeId.Id to vol.HealthStatus
+	volumeIdToHealthStatusMap := make(volumeIdHealthStatusMap, len(queryResult.Volumes))
 
-		if pvc, ok := volumeHandleToPvcMap[vol.VolumeId.Id]; ok {
-			log.Debugf("csiGetVolumeHealthStatus: Found pvc %q for volume %q", pvc, vol.VolumeId.Id)
+	for _, vol := range queryResult.Volumes {
+		volumeIdToHealthStatusMap[vol.VolumeId.Id] = vol.HealthStatus
+		log.Debugf("csiGetVolumeHealthStatus: Volume Id %s has HealthStatus %s",
```

Comment:
This will fill up the logs in a scaled env and could result in quicker log rollover in dev env. Can we remove this log?

Reply:
Done